msal/src/Microsoft.Identity.Client/Telemetry.cs

```diff
@@ -187,7 +193,7 @@ internal void Flush(string requestId)
 
             if (eventsToFlush.Count > 0)
             {
-                eventsToFlush.Insert(0, new DefaultEvent(ClientId));
+                eventsToFlush.Insert(0, new DefaultEvent(ClientId, eventCountToFlush));
```

Comment:
eventCountToFlush could be null because the TryRemove can return false

Reply:
done